spec/models/report_rendering_options_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe ReportRenderingOptions, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I'd delete this file as it contains no spec actually.

Reply:
Done : )